bpf/bpf_lxc.c

```diff
@@ -939,7 +941,7 @@ __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT64) int tail_ipv6_to_ipv4(struct
 
 __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT46) int tail_ipv4_to_ipv6(struct __sk_buff *skb)
 {
-	union v6addr dp = LXC_IP;
+	union v6addr dp = {};
```

Comment:
Right in this case these are not needed but (a) LLVM _should_ optimize these out anyways and (b) I'll do a follow up pass over all the code, not just the pieces I changed and clean some of these small things up. Also I check to see if LLVM really does optimize these away. Thanks.